pkg/machine/applehv/stubber.go

```diff
@@ -346,3 +347,14 @@ func (a AppleHVStubber) PostStartNetworking(mc *vmconfigs.MachineConfig, noInfo
 func (a AppleHVStubber) GetDisk(userInputPath string, dirs *define.MachineDirs, mc *vmconfigs.MachineConfig) error {
 	return diskpull.GetDisk(userInputPath, dirs, mc.ImagePath, a.VMType(), mc.Name)
 }
+
+func (a AppleHVStubber) ConnectionInfo(_ string) (*define.VMFile, *define.VMFile, error) {
+	sockName := "podman.sock"
```

Comment:
This smells like a code duplicate.  I didn't check but is there no other function to get the path to the socket?

Reply:
There used to be `<provider>.forwardSocketPath()` but it looks like that got nuked with other podman 4 code. The duplication will be solved by moving the `ConnectionInfo` function to `MachineConfig` instead